data/json/items/comestibles/drink.json

```diff
@@ -210,26 +197,28 @@
     "id": "con_milk",
     "name": "condensed milk",
     "name_plural": "condensed milk",
-    "weight": 258,
+    "weight": 321,
     "color": "white",
     "spoils_in": "1 day",
     "container": "can_food",
     "comestible_type": "DRINK",
     "symbol": "~",
-    "quench": 20,
-    "calories": 95,
+    "quench": 12,
+    "calories": 103,
     "description": "Baby cow food, appropriated for adult humans.  Having been canned, this milk should last for a very long time.",
     "price": 700,
     "material": [ "milk", "junk" ],
     "volume": 1,
     "phase": "liquid",
+    "charges": 10,
```

Comment:
Because you changed the charges of this item, you need to divide the weight and price by the number of charges.